tests/test_server_info.py

```
@@ -0,0 +1,242 @@
+    """QdrantClient.server_info() delegates to the inner client."""
+
+    def test_local_in_memory_returns_synthetic(self):
+        client = QdrantClient(":memory:")
+        result = client.server_info()
+        assert result is not None
+        assert result.title == "qdrant-client (local mode)"
+        assert isinstance(result.version, str)
+        assert result.commit is None
+
+    def test_local_path_returns_synthetic(self):
+        client = QdrantClient(path="/tmp/qdrant_server_info_test")
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/qdrant_server_info_test"

(S108)